definitions/tools/cellranger_count.cwl

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Run Cell Ranger Count"
+
+baseCommand: ["cellranger", "count", "--localmem=64", "--localcores=8", "--id=cellranger_output"]
+
+requirements:
+    - class: DockerRequirement
+      dockerPull: "registry.gsc.wustl.edu/ebelter/cellranger:2.2.0"
```

Comment:
Can we update this to the 3.0 version that is now out?
`registry.gsc.wustl.edu/mgi/cellranger:3.0.1`

Reply:
I'll update the cwls to 3.0 and run my tests. Can make a new pr to do that so this one can get merged?